### DISCUSSION TITLE
It sucks that the github search box is on the right

### DISCUSSION THREAD

**Discussion by sbfkcel** [community/community#59919] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The search box is a frequently used function of github. People always look for things from the upper left corner to the right and down. And the search box is often blocked by the browser extension window.

Such an important function should be placed on the left as before.

And after the search box gets the focus, it is displayed in full screen again. At this time, the line of sight needs to move from right to left, and the experience is very bad.

**Comment by artemik** [comment]
Confirm. I don't understand how most activities and, for example, the repositories search "Find a repository..." is on the left, while the most used area like global search is far on the right. The repo/page names on the left have different width and that's probably why it's a problem to put the search box there in a fixed place, but the current decision is very bad. Github chose UI vs UX.

**Comment by sbfkcel** [comment]
From my personal habits, I prefer the previous position on the left.

Now I need to move my sight to the right, and when the input gets the focus, it will be displayed in full screen, so my sight will move to the left again. Very bad.

**Comment by davidstaheli** [comment]
@sbfkcel and @artemik, thank you very much for this feedback. We understand your perspective on this and want to let you know that we're giving it full attention. We greatly appreciate your help in making it better.
